python/GangaDirac/Lib/Credentials/DiracProxy.py

```diff
@@ -153,6 +154,11 @@ def default_location(self):
         else:
             return base_proxy_name
 
+    @property
+    def location(self):
+        """
+        """
+        return self.default_location()
```

Comment:
This was orignally to aid in debugging, it just overrides the underlying logic to call the `default_location` directly so I can remove it if required.